std/algorithm/searching.d

```diff
@@ -1268,6 +1268,9 @@ if (isInputRange!R &&
     }}
 }
 
+/// Is `true` iff `T` has a `const` member function named `empty`.
```

Comment:
Comment is not strictly correct. `empty` doesn't need to be a member function (for instance this works with dynamic arrays & infinite ranges) & `empty` needs to be implicitly convertible to bool or have a return type that is implicitly convertible to bool. Also I'm not sure if it matters but maybe we shouldn't use DDoc `///` for `private`.

Reply:
Ok, I remove the comment.